src/js/components/FormField/FormField.js

```diff
@@ -327,7 +327,10 @@ const FormField = forwardRef(
           if (onFocus) onFocus(event);
         }}
         onBlur={event => {
-          setFocus(false);
+          const role = event.target.getAttribute('role');
```

Comment:
This seems like a fragile approach. I think we should look for an alternate approach to addressing this.

Reply:
gotcha, can do.